src/appshell/internal/applicationuiactions.cpp

```diff
@@ -54,10 +54,10 @@ const UiActionList ApplicationUiActions::m_actions = {
              TranslatableString("action", "Full screen"),
              Checkable::Yes
              ),
-    UiAction("about",
+    UiAction("about-musescore",
              mu::context::UiCtxAny,
              mu::context::CTX_ANY,
-             TranslatableString("action", "&About…")
+             TranslatableString("action", "About &MuseScore…")
```

Comment:
Leave the & at the A, ("&About MuseScore…"), it conflicts with "About &MusicXML…" otherwise

Reply:
Done